Civi/Api4/Generic/AbstractEntity.php

```diff
@@ -140,69 +140,73 @@ protected static function getDaoName(): ?string {
    * @return array{name: string, title: string, description: string, title_plural: string, type: string, paths: array, class: string, primary_key: array, searchable: string, dao: string, label_field: string, icon: string}
    */
   public static function getInfo() {
+    static $getInfo;
```

Comment:
We tend to avoid `static` variables because it can't be cleared between test runs. I recommend `Civi::$statics` instead.

Reply:
@mattwire I'm good with this change otherwise